examples/gallery/plot/scatter.py

```diff
@@ -21,19 +21,19 @@
     projection="X10c/10c",
     frame=["xa0.2fg", "ya0.2fg", "WSrt"],
 )
-for color in ["blue", "orange", "green"]:
+for color, style in [("gray73", "c"), ("darkorange", "s"), ("slateblue", "t")]:
     x, y = np.random.rand(2, n)  # random X and Y data in [0,1]
     sizes = np.random.rand(n) * 0.5  # random size [0,0.5], in cm
     # plot data points as circles (style="c"), with different sizes
     fig.plot(
         x=x,
         y=y,
-        style="c",
+        style=style,
         sizes=sizes,
         color=color,
         # Set the legend label,
-        # and set the circle size to be 0.25 cm (+S0.25c) in legend
-        label=f"{color}+S0.25c",
+        # and set the style size to be 0.25 cm (+S0.25{style}) in legend
+        label=f"{color}+S0.25{style}",
         transparency=70,  # set transparency level for all symbols
```

Comment:
```suggestion
        transparency=50,  # set transparency level for all symbols
```
Perhaps 50 gives a better result.

Reply:
Yes, please try 50, i.e. less transparency (more opaque). See also https://github.com/GenericMappingTools/pygmt/pull/1013#discussion_r594010939.